Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -273,6 +274,15 @@ def main():
     pr.create_issue_comment(body)
     print(f'{t.cyan}Created welcome comment{t.normal}')
 
+    print('contributors.md section')
+    print(f'pack path: {pr_files[0]}')
+    ver = get_pack_metadata(pr_files[0]).get('currentVersion')
+    print(f'version is: {ver}')
+    contributors_body = f'Hi @{pr.user.login}, Thanks for contributing to a Cortex XSOAR supported pack. To receive ' \
+                        f'credit for your generous contribution please follow this [link]' \
+                        f'(https://xsoar.pan.dev/docs/packs/packs-format#contributorsjson).'
+    if 'contributors' not in pr_files and XSOAR_SUPPORT_LEVEL_LABEL in labels_to_add and ver != '1.0.0':
```

Comment:
There is no need to check `if 'contributors' not in pr_files`. since this file can exist already in the pack but without the current contributor.
```suggestion
    if XSOAR_SUPPORT_LEVEL_LABEL in labels_to_add and ver != '1.0.0':
```

Reply:
correct, changed